src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -120,7 +121,7 @@
    (fn [{:keys [state props refs]}]
      [:div {}
       (when (:show-import-mc-modal? @state)
-        (render-import-modal state props refs))
+        (render-import-modal state props refs (:on-import props)))
```

Comment:
Looks like my previous comments got blasted by the force-push.  This doesn't need to be pulled out of props here, you can leave it in and then extract it on line 113.